File: 백준/Bronze/1977. 완전제곱수/완전제곱수.cc
#include <iostream>
using namespace std;

int main() {
	
	int minN;
	int maxN;
	cin >> minN >> maxN;

	int sum = 0;
	int minSquareN = maxN;
	for (int i = 1; i * i <= maxN; i++)
	{
		if (i * i >= minN && i * i <= maxN)
		{
			if (i * i <= minSquareN)
			{
				minSquareN = i * i;
			}

			sum += i * i;
		}
	}

	if (sum == 0)
	{
		cout << -1 << endl;
	}
	else {
		cout << sum << '\n' << minSquareN << endl;

	}
}

File: 백준/Bronze/1977. 완전제곱수/완전제곱수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int minN = sc.nextInt();
        int maxN = sc.nextInt();

        int sum = 0;
        int minSquareN = maxN;

        for (int i = 1; i * i <= maxN; i++) {
            if (i * i >= minN &&  i * i <= maxN) {
                sum += i * i;

                if (i * i <= minSquareN) {
                    minSquareN = i * i;
                }
            }
        }

        if (sum == 0) {
            System.out.println(-1);
        } else {
            System.out.println(sum);
            System.out.println(minSquareN);
        }
    }
}